packages/vue/src/components/primitives/base-form-field.vue

```diff
@@ -23,8 +24,14 @@ const { validationErrors } = toRefs(useAuthShared());
 
 const { type } = formField.value;
 
-const errors = computed(() => getErrors(validationErrors.value[name.value]));
 const isPasswordField = type === 'password';
+
+const errors = computed(() => getErrors(validationErrors.value[name.value]));
+const hasError = computed(() => errors.value?.length > 0);
+const errorId = computed(() => nanoid(12));
```

Comment:
Does the nanoid ever change? Else you don't need this in a computed...

Reply:
Ooh, good catch!

Reply:
c06c77404ca847c4072fff0ea1577d986808d4d1